modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -14,11 +14,26 @@ In addition to using Anypoint Runtime Manager, to deploy applications to CloudHu
 // tag::prerequisites[]
 == Prerequisites
 
-* The Mule Maven Plugin is added to your project
+* The Mule Maven Plugin (version 3.8.1) is added to your project
 +
 See xref:mmp-concept.adoc#add-mmp[Add the Mule Maven Plugin to a Mule Project] for instructions.
 * The application is already published in Exchange +
 See xref:exchange::to-publish-assets-maven.adoc[Publish and Deploy Exchange Assets Using Maven].
+* The Mule Maven Facade API (v3) is added as a repository in the distribution management section of your project's POM file. For example: 
++
+[source,xml]
+----
+  <distributionManagement>
+    <repository>
+      <id>Repository</id>
+      <name>Corporate Repository</name>
+      <url>https://maven.anypoint.mulesoft.com/api/v3/organizations/ORGANIZATION_ID/maven</url>
+      <layout>default</layout>
+    </repository>
+  </distributionManagement>
+----
++
+See https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[The Mule Maven Facade API documentation] for more information.
```

Comment:
```suggestion
See https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/exchange-maven-facade-api-http/[^The Mule Maven Facade API documentation] for more information.
```

Reply:
I forget which side the caret goes on for external links. I think it's left side of the page description.